docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,84 @@
+= Parameters
+
+The parent key for all of the following parameters is `cortex_tenant_ns_label`.
+
+== `app_name`
+
+[horizontal]
+type:: string
+default:: `cortex-tenant-ns-label`
+
+Configure the deployment name of the cortex-tenant-ns-label reverse proxy.
```

Comment:
I don't see a real reason why users might want to change the deployment name, but it doesn't really hurt to have it either.